test/views/empty-state.spec.js

```diff
@@ -73,6 +79,7 @@ describe('Component:  pfEmptyState', function () {
     expect(element.find('.blank-state-pf-helpLink').text()).toContain('For more information please see');
     expect(element.find('a').text()).toContain('pfExample');
     expect(element.find('a').prop('href')).toContain('#/api/patternfly.views.component:pfEmptyState');
+    expect(element.find('.blank-state-pf-helpLink').click().length).toBe(1);
```

Comment:
I think a better way to test callback functions in unit tests would be:

```
element.find('.blank-state-pf-helpLink').click();
expect($scope.eventText).toBe('Empty State Action Executed');
```

Reply:
Gotcha! I had something similar to that initially, thanks Dave! :)